src/deploy/functions/params.ts

```diff
@@ -123,6 +143,12 @@ export function isSelectInput<T>(input: ParamInput<T>): input is SelectInput<T>
 export function isResourceInput<T>(input: ParamInput<T>): input is ResourceInput {
   return {}.hasOwnProperty.call(input, "resource");
 }
+/**
+ * Determines whether an Input field value can be coerced to ListSelectInput.
```

Comment:
The documentation and function name doesn't seem to match

```suggestion
 * Determines whether an Input field value can be coerced to MultiSelectInput.
```